device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/DeviceClientTest.java

```diff
@@ -11,6 +11,7 @@
 import com.microsoft.azure.sdk.iot.device.fileupload.FileUpload;
 import com.microsoft.azure.sdk.iot.device.transport.amqps.IoTHubConnectionType;
 import com.microsoft.azure.sdk.iot.provisioning.security.SecurityProvider;
+import com.sun.security.ntlm.Client;
```

Comment:
What is this import for?

Reply:
I'll remove it, it may have been added when I was adding some changes.